docs/source/getting-started.rst

```diff
@@ -0,0 +1,89 @@
+===============
+Getting Started
+===============
+
+This section provides an introduction on how to get started with using the SymbiFlow toolchain.
+In order to generate a bitstream (or any intermediate file format),
+you can use one of the toolchain tests. Following steps describe the whole
+process:
+
+Clone repository
+----------------
+
+.. code-block:: bash
+
+    git clone --recursive https://github.com/SymbiFlow/symbiflow-arch-defs.git
+
+Install dependencies
+--------------------
+
+.. code-block:: bash
+
+    sudo apt install libxml2-dev libxslt1-dev lib32z1-dev
+
+    cd symbiflow-arch-defs
+    pip3 install -r requirements.txt
+
+Prepare environment
+-------------------
+
+Download all needed packages and databases content
+into the separated environment
+
+.. code-block:: bash
+
+    make env
+
+Build example
+-------------
+
+Enter the appropriate test build directory, depending on your target
+architecture and invoke the appropriate make target.
+
+Build directories depend on the architecture. Because of that,
+depending on the chosen target, a different toolchain backend will be used.
+(See `Project X-Ray <https://prjxray.readthedocs.io/en/latest/>`_
+and `Project Trellis <https://prjtrellis.readthedocs.io/en/latest/>`_
+for more information)
+
+Moreover, it is worth to note that target names have the form <*testname_platform_outputformat*>.
+
+Assuming that you would like to generate bitstream ``.bit`` file with
+the counter example for the arty board, which uses Xilinx Artix-7 FPGA,
+you will type:
+
+.. code-block:: bash
+
+    cd build/xc7/tests/counter
+    make counter_arty_bin
+
+Load bitstream
+--------------
+
+The last step is to load the bitstream to your platform.
```

Comment:
The `_prog` target (e.g. `make counter_arty_prog`) should work.  So manually finding and programming the bitstream should not be required.

Reply:
I think it's not always the case. The Makfile target for _something_prog is mostly empty.
Maybe this is a bug?
Anyway I added a ``make counter_arty_prog`` to the doc, but left the description of how to do it manually for now.

Reply:
Bug, it should work. Can you file an issue?

Reply:
Will do